getting-started/README.adoc

```diff
@@ -170,6 +171,60 @@ ifndef::qs[]
 . Verify that your service account was successfully created in the *Streams for Apache Kafka* > *Service Accounts* page.
 endif::[]
 
+[id="proc-setting-service-account-permissions_{context}"]
+== Setting permissions for a service account in a Kafka instance in {product}
+
+{product} uses Access Control Lists (ACLs) provided by Kafka that enable you to manage how other user accounts and service accounts are permitted to interact with the Kafka resources that you create. When you create a service account to connect to a Kafka instance, you must also set the appropriate level of permissions for that new account in the ACL of the instance.
+
+The permissions described in this procedure enable applications associated with the service account to fully interact with the resources in the instance.
+
+.Prerequisites
+* You've created a Kafka instance and the instance is in *Ready* state.
+* You've created a service account that you want to allow to interact with the running Kafka instance.
+
+.Procedure
+. In the *Kafka Instances* page of the web console, click the name of the Kafka instance that you want the service account to access.
+. Click the *Permissions* tab to view the current ACL for this instance.
+. Click *Manage access*, use the *Account* drop-down menu to select the service account that you previously created, and click *Next*.
+. Under *Assign Permissions*, use the drop-down menus to set the following permissions for the specified resource types for this service account. Click *Add* to add each new resource permission.
++
+--
+These permissions enable applications associated with the service account to produce and consume messages in any topic in the instance, using any consumer group and any producer.
+
+.ACL permissions for a new service account
```

Comment:
I would add `Example.. `

Reply:
Sure. I intentionally didn't say "Example" here but did say that in the access mgmt doc, because here we're prescriptively telling the user to just do this literally, not do something like this, where it's just an example. But I do see the benefit of labeling as such for consistency. Thanks.